Source/OxyPlot/Series/ContinuousHistogramHelpers.cs

```diff
@@ -0,0 +1,117 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="ContinuousHistogramHelpers.cs" company="OxyPlot">
+//   Copyright (c) 2014 OxyPlot contributors
+// </copyright>
+// <summary>
+//   Provides methods to collect data samples into bins for use with a <see cref="ContinuousHistogramSeries" />.
+// </summary>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace OxyPlot.Series
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    
+    /// <summary>
+    /// Provides methods to collect data samples into bins for use with a <see cref="ContinuousHistogramSeries" />.
+    /// </summary>
+    public static class ContinuousHistogramHelpers
```

Comment:
Is this a builder? https://en.wikipedia.org/wiki/Builder_pattern

Reply:
Should it be a non-static class? I think I would prefer that, even if it is a single call... Not sure...

Reply:
I didn't intend to make it a builder: it is just data processing really (data which can be fed directly to the series). I contemplated making it just part of the `ContinousHistogramItem` series, but I couldn't find a real precedent either way in the codebase.